<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DGeeClassifierManager</class>
 <widget class="QWidget" name="DGeeClassifierManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DGeeClassifierManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	outline:0px;
	color:#DCDCDC;
}

*:disabled{
	background:#444444;
	border-color:#484848;
}

QWidget{
	border:1px solid #DCDCDC;
	background:#484848;
	border-radius:0px;
	color:#DCDCDC;
	border-style:none;
}

QSplitter::handle{
background:#7c7c7c;
}

QAbstractButton{
	border-radius:0px;
	color:#DCDCDC;
	background:none;
	border-style:none;
}

QAbstractButton:hover{
	color:#FFFFFF;
	background-color:#00BB9E;
}

QAbstractButton:checked,QAbstractButton:pressed{
	color:#DCDCDC;
	border-style:solid;
	border-width:0px 0px 0px 2px;
	padding:4px 4px 4px 2px;
	border-color:#00BB9E;
	background-color:#444444;
}

QLineEdit{
	border:1px solid #242424;
	border-radius:3px;
	padding:2px;
	background-color: rgb(90, 90, 90);
	color:#DCDCDC;
	selection-background-color:#484848;
	selection-background-color: rgba(81, 163, 244);
}

QLineEdit:focus{
	border:1px solid #242424;
}

QLineEdit{
	lineedit-password-character:9679;
}

.QPushButton{
	border-style:none;
	border:1px solid #242424;
	color:#DCDCDC;
	padding:5px;
	min-height:15px;
	border-radius:5px;
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

.QPushButton:hover{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

.QPushButton:pressed{
	background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QComboBox{
	border:1px solid #8f8f8f;
	background-color: rgb(118, 118, 118);
	border-radius: 5px;
}

QComboBox::down-arrow{
	image:url(:/images/qss/psblack/add_bottom.png);
	width:10px;
	height:10px;
	right:2px;
}

QComboBox::drop-down{
	subcontrol-origin:padding;
	subcontrol-position:top right;
	width:15px;
	border-left-width:0px;
	border-left-style:solid;
	border-top-right-radius:3px;
	border-bottom-right-radius:3px;
	border-left-color:#242424;
}

QTabBar::tab{
border:1px solid #242424;
color:#DCDCDC;
margin:0px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #646464,stop:1 #525252);
}

QTabBar::tab:selected,QTabBar::tab:hover{
border-style:solid;
border-color:#00BB9E;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #484848,stop:1 #383838);
}

QTabBar::tab:top,QTabBar::tab:bottom{
padding:3px 8px 3px 8px;
}

QTabBar::tab:left,QTabBar::tab:right{
padding:8px 3px 8px 3px;
}

QTabBar::tab:top:selected,QTabBar::tab:top:hover{
border-width:2px 0px 0px 0px;
}

QTabBar::tab:right:selected,QTabBar::tab:right:hover{
border-width:0px 0px 0px 2px;
}

QTabBar::tab:bottom:selected,QTabBar::tab:bottom:hover{
border-width:0px 0px 2px 0px;
}

QTabBar::tab:left:selected,QTabBar::tab:left:hover{
border-width:0px 2px 0px 0px;
}

QTabBar::tab:first:top:selected,QTabBar::tab:first:top:hover,QTabBar::tab:first:bottom:selected,QTabBar::tab:first:bottom:hover{
border-left-width:1px;
border-left-color:#242424;
}

QTabBar::tab:first:left:selected,QTabBar::tab:first:left:hover,QTabBar::tab:first:right:selected,QTabBar::tab:first:right:hover{
border-top-width:1px;
border-top-color:#242424;
}

QTabBar::tab:last:top:selected,QTabBar::tab:last:top:hover,QTabBar::tab:last:bottom:selected,QTabBar::tab:last:bottom:hover{
border-right-width:1px;
border-right-color:#242424;
}

QTabBar::tab:last:left:selected,QTabBar::tab:last:left:hover,QTabBar::tab:last:right:selected,QTabBar::tab:last:right:hover{
border-bottom-width:1px;
border-bottom-color:#242424;
}

QComboBox::drop-down:on{
	top:1px;
}

QTableView[model=&quot;true&quot;]::item{
padding:0px;
margin:0px;
}

QTableView QLineEdit,QTableView QComboBox,QTableView QSpinBox,QTableView QDoubleSpinBox,QTableView QDateEdit,QTableView QTimeEdit,QTableView QDateTimeEdit{
border-width:0px;
border-radius:0px;
}

QTableView QLineEdit:focus,QTableView QComboBox:focus,QTableView QSpinBox:focus,QTableView QDoubleSpinBox:focus,QTableView QDateEdit:focus,QTableView QTimeEdit:focus,QTableView QDateTimeEdit:focus{
border-width:0px;
border-radius:0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="mLabelLayerName">
       <property name="text">
        <string>Gee Layer*:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mLineEditLayerName">
       <property name="placeholderText">
        <string>eeObj variable Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mComBoxLayerName"/>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnSummitLayer">
       <property name="text">
        <string>Summit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="mLabelSamples">
       <property name="text">
        <string>Samples*:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mLineEditSamplesName">
       <property name="placeholderText">
        <string>FeatureCollection variable Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mComBoxSamples"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="mLabelROI">
       <property name="text">
        <string>ROI:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mLineEditRoi">
       <property name="placeholderText">
        <string>Roi variable Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mComBoxRois"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QComboBox" name="mComboxCanbeSelectBands"/>
     </item>
     <item>
      <widget class="QLabel" name="mLabelArrow">
       <property name="text">
        <string>&lt;=&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mComBoxSelectBands"/>
     </item>
     <item>
      <widget class="QLabel" name="mLabelROI_3">
       <property name="text">
        <string>Scale:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mLineEditScale">
       <property name="placeholderText">
        <string>Scale</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="mLabelClassifier">
       <property name="text">
        <string>Classifier:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mComboxClassifier"/>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnSetClassParams">
       <property name="text">
        <string>SetParams</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnTrain">
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnExplain">
       <property name="text">
        <string>Explain</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="mLineEditTrainedModel">
       <property name="placeholderText">
        <string>Trained Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnClassify">
       <property name="text">
        <string>Classify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnTrainAndClassify">
       <property name="text">
        <string>QuickClassify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnConfusionTable">
       <property name="text">
        <string>Precison</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mTabConfusionMatrix">
      <attribute name="title">
       <string>Matrix</string>
      </attribute>
      <widget class="QTableView" name="mTableViewConfusionTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>371</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mTabDetails">
      <attribute name="title">
       <string>DeTails</string>
      </attribute>
      <widget class="QTextEdit" name="mTextEditDetails">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>421</width>
         <height>261</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mBtnShowParams">
       <property name="text">
        <string>ShowParams</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mLineEditLayerName_2">
       <property name="placeholderText">
        <string>Layer Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnAddToMap">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mBtnQuit">
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
